core-infrastructure/src/db/Core.Database/Scripts/006-CreateFinancialPlanTable.sql

```diff
@@ -0,0 +1,21 @@
+IF NOT EXISTS(SELECT *
+              FROM INFORMATION_SCHEMA.TABLES
+              WHERE table_name = 'FinancialPlan')
+    BEGIN
+        CREATE TABLE dbo.FinancialPlan
+        (
+            UKPRN          nvarchar(50)   NOT NULL,
```

Comment:
Would it be better to define the UKPRNs as integers

Reply:
Keeping it flexible for the moment, as we discussion be might be to change this in future.